--- iodine-0.7.0/Makefile.orig	2015-04-03 21:06:29.560818850 +0200
+++ iodine-0.7.0/Makefile	2015-04-03 21:13:53.784142924 +0200
@@ -17,7 +17,7 @@
 TARGETOS = `uname`
 
 all:
-	@(cd src; $(MAKE) TARGETOS=$(TARGETOS) all)
+	$(MAKE) -C src TARGETOS=$(TARGETOS) all
 
 install: all
 	$(MKDIR) $(MKDIR_FLAGS) $(DESTDIR)$(sbindir)
@@ -37,12 +37,12 @@
 test: all
 	@echo "!! The check library is required for compiling and running the tests"
 	@echo "!! Get it at http://check.sf.net"
-	@(cd tests; $(MAKE) TARGETOS=$(TARGETOS) all)
+	$(MAKE) -C tests TARGETOS=$(TARGETOS) all
 
 clean:
 	@echo "Cleaning..."
-	@(cd src; $(MAKE) clean)
-	@(cd tests; $(MAKE) clean)
+	$(MAKE) -C src clean
+	$(MAKE) -C tests clean
 	@rm -rf bin iodine-latest*
 
 #Helper target for windows/android zipfiles
