--- iodine-0.7.0/src/Makefile.orig	2015-04-03 21:04:15.740821639 +0200
+++ iodine-0.7.0/src/Makefile	2015-04-03 21:06:13.590819177 +0200
@@ -8,8 +8,9 @@
 ARCH = `uname -m`
 
 LIBPATH = -L.
+OPTFLAGS = -g -Wall
 LDFLAGS +=  -lz `sh osflags $(TARGETOS) link` $(LIBPATH)
-CFLAGS += -std=c99 -c -g -Wall -D$(OS) -pedantic `sh osflags $(TARGETOS) cflags`
+CFLAGS += -std=c99 -c $(OPTFLAGS) -D$(OS) -pedantic
 
 all: stateos $(CLIENT) $(SERVER)
 
