--- iodine-0.4.2/Makefile	2008-11-21 13:44:41.755943542 +0200
+++ iodine-0.4.2/Makefile	2008-11-21 13:45:21.234222315 +0200
@@ -1,4 +1,4 @@
-prefix=/usr/local
+prefix=/usr
 sbindir=$(prefix)/sbin
 datadir=$(prefix)/share
 mandir=$(datadir)/man
@@ -15,7 +15,7 @@
 RM_FLAGS=-f
 
 all: 
-	@(cd src; $(MAKE) all)
+	$(MAKE) -C src all
 
 install: all
 	$(MKDIR) $(MKDIR_FLAGS) $(DESTDIR)$(sbindir)
@@ -36,10 +36,10 @@
 	@echo "!! The check library is required for compiling and running the tests"
 	@echo "!! Get it at http://check.sf.net"
-	@(cd tests; $(MAKE) all)
+	$(MAKE) -C tests all
 
 clean:
 	@echo "Cleaning..."
-	@(cd src; $(MAKE) clean)
-	@(cd tests; $(MAKE) clean)
+	@$(MAKE) -C src clean
+	@$(MAKE) -C tests clean
 	@rm -rf bin
 
