--- iodine-0.4.0/Makefile	2007-06-09 01:12:08.835961750 +0300
+++ iodine-0.4.0/Makefile	2007-06-09 01:13:12.717415147 +0300
@@ -6,30 +6,31 @@
 MKDIR=mkdir
 MKDIR_FLAGS=-p
+MANDIR=$(PREFIX)/man
 
 RM=rm
 RM_FLAGS=-f
 
 all: 
-	@(cd src; make all)
+	$(MAKE) -C src all
 
 install: all
-	$(MKDIR) $(MKDIR_FLAGS) $(PREFIX)/sbin
-	$(INSTALL) $(INSTALL_FLAGS) bin/iodine $(PREFIX)/sbin/iodine
-	$(INSTALL) $(INSTALL_FLAGS) bin/iodined $(PREFIX)/sbin/iodined
-	$(MKDIR) $(MKDIR_FLAGS) $(PREFIX)/man/man8
-	$(INSTALL) $(INSTALL_FLAGS) man/iodine.8 $(PREFIX)/man/man8/iodine.8
+	$(MKDIR) $(MKDIR_FLAGS) $(DESTDIR)$(PREFIX)/sbin
+	$(INSTALL) $(INSTALL_FLAGS) bin/iodine $(DESTDIR)$(PREFIX)/sbin/iodine
+	$(INSTALL) $(INSTALL_FLAGS) bin/iodined $(DESTDIR)$(PREFIX)/sbin/iodined
+	$(MKDIR) $(MKDIR_FLAGS) $(DESTDIR)$(MANDIR)/man8
+	$(INSTALL) $(INSTALL_FLAGS) man/iodine.8 $(DESTDIR)$(MANDIR)/man8/iodine.8
 
 uninstall:
-	$(RM) $(RM_FLAGS) $(PREFIX)/sbin/iodine
-	$(RM) $(RM_FLAGS) $(PREFIX)/sbin/iodined
-	$(RM) $(RM_FLAGS) $(PREFIX)/man/man8/iodine.8
+	$(RM) $(RM_FLAGS) $(DESTDIR)$(PREFIX)/sbin/iodine
+	$(RM) $(RM_FLAGS) $(DESTDIR)$(PREFIX)/sbin/iodined
+	$(RM) $(RM_FLAGS) $(DESTDIR)$(PREFIX)/man/man8/iodine.8
 	
 test: all
-	@(cd tests; make all)
+	$(MAKE) -C tests all
 
 clean:
 	@echo "Cleaning..."
-	@(cd src; make clean)
-	@(cd tests; make clean)
+	@$(MAKE) -C src clean
+	@$(MAKE) -C tests clean
 	@rm -rf bin
 
