--- iodine-0.7.0/src/osflags~	2014-06-16 22:28:43.000000000 +0200
+++ iodine-0.7.0/src/osflags	2017-02-26 00:19:38.951204153 +0100
@@ -19,7 +19,7 @@
 		Linux)
 			FLAGS="";
 			[ -e /usr/include/selinux/selinux.h ] && FLAGS="$FLAGS -lselinux";
-			[ -e /usr/include/systemd/sd-daemon.h ] && FLAGS="$FLAGS -lsystemd-daemon";
+			[ -e /usr/include/systemd/sd-daemon.h ] && FLAGS="$FLAGS -lsystemd";
 			echo $FLAGS;
 		;;
 	esac
